packages/traceloop-sdk/traceloop/sdk/tracing/tracing.py

```diff
@@ -865,3 +874,21 @@ def init_weaviate_instrumentor():
         logging.warning(f"Error initializing Weaviate instrumentor: {e}")
         Telemetry().log_exception(e)
         return False
+
+
+def init_alephalpha_instrumentor():
+    try:
+        if importlib.util.find_spec("alephalpha") is not None:
```

Comment:
Isn't the name of the package `alephalpha-client`?

Reply:
Oversaw that, my bad. I changed it to "aleph_alpha_client".